_i18n/en/general/privacy.md

```diff
@@ -5,30 +5,48 @@ The AntennaPod app only stores data strictly necessary to for the app to functio
   The storage permission is used to open files that are not downloaded directly from within the app. It can also be used to import app settings or OPML files. The permission is only requested if you perform an action that requires reading from storage.
 
+## What data other parties might collect, store and process
+
+- Google: If you have activated backup & reset in your phone settings (Settings → Backup & Reset → Back up my data), you should be aware that Android itself will periodically save a copy of your phone's data in Google's servers. This backup contains private information, including your WiFi passwords, messages and call history. It may also include data from AntennaPod and from other apps you use. The developers of AntennaPod do not have access to this data. For more information, see [Google's privacy policy](https://policies.google.com).
+- Podcast hosters: The web servers that provide the podcast feeds might collect additional data, such as your IP address, access time and what is being accessed. This includes the episodes you are downloading or streaming. Please refer to their respective privacy policies for details. You can view the URL of a podcast by opening the podcast and pressing the info icon. AntennaPod does not allow remote servers to set cookies. The servers can detect the fact that you are using AntennaPod and which version of AntennaPod you are using (HTTP User-Agent). If multiple podcasts are hosted on the same server, the server can detect the list of podcasts that you subscribed to and that are hosted on that server. This might happen if publishers use feedburner, podtrac or similar services to distribute their feeds.
+- Search services: When using the search feature in AntennaPod, the services PodcastIndex.org, iTunes, fyyd and gpodder.net may store your query (search term, IP address, app name "AntennaPod"). After subscribing, the feed is usually served from the creator's server, meaning the search service is no longer involved.
+- Synchronisation services: When enabled via the settings, AntennaPod synchronises your data. This data may include subscribed podcasts, listened episodes, play, pause & favorite actions with date & time stamps and IP address. For more information, see your provider's privacy policy (e.g. [gpodder.net's privacy policy](https://gpodder.net/privacy), or the one of your Nextcloud host).
+- Open Collective (Open Collective Inc, USA; Open Collective Europe ASBL, Belgium): When you make a donation, these entities get access to certain data. For more information, see [Open Collective's privacy policy](https://opencollective.com/privacypolicy).
+
+## What data the AntennaPod core team may have access to
+The developers of AntennaPod do **not** have access to any of your information, except when you
+
+- actively share debugging information (either via email using the 'Crash Report' function, or via the Android crash reporting functionality of Google):
+  - device type
```

Comment:
Some of the items use capital letters and some don't. I think it would be more consistent to make them all capital.

Reply:
In this section, none of the items use capital letters, unless it concerns a brand name (Android, AntennaPod). I opted for small letters because it's technically a continuation of the sentence (except when you: actively share information). Or did I misunderstood?

Reply:
Hmm, right. I think it somehow looks weird to use lower case letters for bullet points but it is the correct grammar.